src/main/java/org/mskcc/cbio/oncokb/web/rest/slack/DropdownEmailOption.java

```diff
@@ -292,6 +303,11 @@ public DropdownEmailOptionBuilder isNotModalEmail() {
             return this;
         }
 
+        /**
+         *
+         * @param modalTitle Title showing in the slack modal. It cannot be longer than 15 characters.
```

Comment:
Is the 15 characters a hard requirement? Most of our titles are over 15 character.

Reply:
Sorry, it should be 25 characters. And it's a hard requirement.